docs/using/downloading.md

```diff
@@ -0,0 +1,232 @@
+}   
+```
+
+The download URL for the full product package is provided near the end of the response for each job in the `jobs` 
+array of the response JSON. The value is paired with the `url` key in the `files` array. 
+
+You can copy and paste each product URL directly into a browser window, or script a workflow to pull all the 
+product URLs from the response JSON into a bulk download function. It may be more convenient to use the 
+[HyP3 Python SDK](#accessing-products-using-the-hyp3-python-sdk) to script bulk download functionality.
+<!--[TODO: add any relevant info or tips for programmatic approaches to working with the API]-->
+
+### Accessing Products using the HyP3 Python SDK
+
+The [HyP3 Python SDK](../using/sdk.md) is a Python wrapper around the HyP3 API, and provides convenient search 
+and download functionality for On Demand products (HyP3 jobs). 
+[This sample notebook](https://github.com/ASFHyP3/hyp3-sdk/blob/main/docs/sdk_example.ipynb){target=_blank} 
+demonstrates how to use the SDK for a range of workflows. 
+
+Use the `find_jobs` method from the `HyP3` class to generate a list of products to download (batch), then use the 
+`download_files` method from the `Batch` class to download all the products in the list. Refer to the 
+[HyP3 SDK API Reference](https://hyp3-docs.asf.alaska.edu/using/sdk_api/#hyp3_sdk ){target=_blank} 
+for more information. 
+
+## Product Packaging and Extraction
+
+On Demand products from ASF are delivered as zip files. The files contained in the zip archive vary by product type, 
+but the zip archive always includes an internal directory containing all the individual files. The directory names 
+can be quite long, and some users (particularly those using a Windows operating system) will need to make 
+accommodations in order to successfully extract the contents.
+
+### Extracting Product Packages
+
+When extracting the contents of a HyP3-generated zip file, you may need to specify a destination directory to prevent 
+the extraction of the internal directory to a directory named with the full zip file name. For many of the products, 
+this combination of directories would result in paths that are longer than can be used with Windows operating systems. 
+
+### Downloading Individual Products
+
+Downloading the full zip file ensures that you have all of the data products as well as auxiliary files and 
+relevant metadata, but some users may not require all of the files included in the product zip archive. 
+
+The contents of the zip files can all be accessed directly. Simply replace the `.zip` at the end of the download URL 
+with the tag for the specific file you want to download. 
+
+For example, for the following download URL for an RTC On Demand product: 
+`https://d3gm2hf49xd6jj.cloudfront.net/76b1a849-c826-428a-966c-55f8bb88f814/S1A_IW_20250502T135654_DVP_RTC30_G_gpuned_70DD.zip`
+
+simply replace the .zip with the product extension, such as the following for the RTC GeoTIFF in VV polarization: 
```

Comment:
I think making clear that `.zip` -> `_VV.tif` is helpful. Otherwise the `_VV.tif` kinda get's lost in the URL.
```suggestion
Replace the `.zip` with the product extension `_VV.tif`. This is the product extension for the RTC GeoTIFF in VV polarization: 
```

Reply:
addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/590/commits/d4045b4ac3e3d1b8fe284f9924df6e94d2dd7ab3)